src/Koffee/MainView.fs

```diff
@@ -464,18 +464,18 @@ module MainStatus =
     let undoingCopy (item: Item) isDeletionPermanent =
         let undoVerb = if isDeletionPermanent then "Deleting" else "Recycling"
         Busy <| sprintf "Undoing copy of %s - %s..." item.Description undoVerb
-    let undoAction action pathFormat repeatCount =
+    let undoAction action pathFormat repeatCount repeatIteration =
         match repeatCount with
         | Some 1 | None -> Message <| (actionCompleteMessage action pathFormat |> sprintf "Action undone: %s")
-        | Some count -> Message <| (actionCompleteMessage action pathFormat |> sprintf "%i actions undone. Last: %s" count)
+        | Some count -> Message <| (actionCompleteMessage action pathFormat |> sprintf "%i/%i actions undone. Last: %s" repeatIteration count)
```

Comment:
As I mentioned above, the status will be set to Busy statuses during the undo operations. The only "resulting" status that the user will actually be able to see is the last one, so this message only needs the total count of actions undone like it was before this commit.